src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -61,6 +72,7 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
                 models_dir, "config.json"
             )
         },
+        m_generation_config{models_dir / "generation_config.json"},
```

Comment:
should not you use `utils::from_config_json_if_exists` helper? because `generation_config.json` can be absent for some pipelines and current line will lead to exception

Reply:
Thanks, fixed